runtimes/common/src/constants.rs

```diff
@@ -324,7 +324,16 @@ pub mod did {
 	/// The size is checked in the runtime by a test.
 	pub const MAX_DID_BYTE_LENGTH: u32 = 9918;
 
-	pub const DID_DEPOSIT: Balance = deposit(2 + MAX_NUMBER_OF_SERVICES_PER_DID, MAX_DID_BYTE_LENGTH);
+	/// The length of all used Keys.
+	pub const KEY_DID_BYTE_LENGTH: u32 = 32;
+
+	pub const MAX_SERVICE_ENDPOINT_BYTE_LENGTH: u32 =
+		MAX_SERVICE_URL_LENGTH + MAX_SERVICE_ID_LENGTH + MAX_SERVICE_TYPE_LENGTH;
+
+	pub const DID_BASE_DEPOSIT: Balance = deposit(2 + MAX_NUMBER_OF_SERVICES_PER_DID, MAX_DID_BYTE_LENGTH);
```

Comment:
Not sure if we should take the MAX_DID_BYTE_LENGHT here. Maybe we should calculate the `MIN_DID_DEPOSIT` and 2 KILT and always take the maximum for the `DID_BASE_DEPOSIT`

Reply:
Could you provide more information, please? The `MIN_DID_DEPOSIT` would be around 0.05 KILTS. I do not fully understand that point.